jupyterlab/labbox_ephys_widgets_jp/src/extensions/devel/IndividualUnits/IndividualUnit.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
     width: number
     calculationPool: CalculationPool
     sortingInfo: SortingInfo
-    sortingUnitViews: {[key: string]: SortingUnitViewPlugin}
+    plugins: Plugins
```

Comment:
I simplified and renamed how registered plugins get passed around. Prior to the change, extensionContext meant two different things -- it referred to the object passed to the extensions that could be used to register plugins, and it also referred to the redux state that contained the registered plugins. I changed the latter to 'plugins' instead. And now 'plugins' state gets passed around as opposed to separately passing around SortingUnitViewPlugins, SortingViewPlugins, etc.